src/OscapScannerRemoteSsh.cpp

```diff
@@ -107,10 +107,28 @@ void OscapScannerRemoteSsh::evaluate()
         return;
     }
 
-    emit infoMessage(QObject::tr("Querying capabilities on remote machine..."));
-
     {
         SshSyncProcess proc(mSshConnection, this);
+        emit infoMessage(QObject::tr("Checking if oscap is available on remote machine..."));
+
+        proc.setCommand(QString("hash"));
```

Comment:
I think people mostly use `which` when they want to be compatible. It's in coreutils.